### PR TITLE
[Linux] Minor documentation improvements

### PR DESCRIPTION
**Pull request by maxim-lobanov**
# Description

Included changes:
1) Remove apt repos for some software from documentation because
   - We don't really need to specify apt repos for tools where we use official recommended installation way
   - It complicates reading docs
   - It doesn't play well with new release notes format.
2) `MSBuild 16.10.1.31701 (from /usr/lib/mono/msbuild/15.0/bin/MSBuild.dll)` -> `MSBuild 16.10.1.31701 (Mono 6.12)`
3) Polish notes for some database tools
4) Change report title to be `Ubuntu 22.04`
5) Add `OS Version:` and rename `Kernel Version:` in report to be consistent with other images

## Check list
- [ ] Related issue / work item is attached
- [ ] Tests are written (if applicable)
- [ ] Documentation is updated (if applicable)
- [ ] Changes are tested and related VM images are successfully generated

### CODE REVIEW

File: images/linux/scripts/SoftwareReport/SoftwareReport.Generator.ps1 [hunk 1]
```diff
@@ -42,8 +40,8 @@ $runtimesList = @(
     (Get-DashVersion),
     (Get-CPPVersions),
     (Get-FortranVersions),
-    (Get-MsbuildVersion),
-    (Get-MonoVersion),
+    "MSBuild $(Get-MsbuildVersion)",
```
Comment (bogdan-damian-bgd) [inline images/linux/scripts/SoftwareReport/SoftwareReport.Generator.ps1:43]: 'MSBuild' is duplicate?

**Review by bogdan-damian-bgd — APPROVED** [review]

File: images/linux/scripts/SoftwareReport/SoftwareReport.Generator.ps1 (hunk 1 above)
Comment (maxim-lobanov) [reply images/linux/scripts/SoftwareReport/SoftwareReport.Generator.ps1]: Thanks, fixed!

**Review by igorboskovic3 — APPROVED** [review]

**Review by mikhailkoliada — APPROVED** [review]

**Pull request merged**